pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -211,29 +211,31 @@ const (
-	CSVReasonAPIServiceResourcesNeedReinstall ConditionReason = "APIServiceResourcesNeedReinstall"
-	CSVReasonAPIServiceInstallFailed          ConditionReason = "APIServiceInstallFailed"
-	CSVReasonCopied                           ConditionReason = "Copied"
-	CSVReasonInvalidInstallModes              ConditionReason = "InvalidInstallModes"
-	CSVReasonNoTargetNamespaces               ConditionReason = "NoTargetNamespaces"
-	CSVReasonUnsupportedOperatorGroup         ConditionReason = "UnsupportedOperatorGroup"
-	CSVReasonNoOperatorGroup                  ConditionReason = "NoOperatorGroup"
-	CSVReasonTooManyOperatorGroups            ConditionReason = "TooManyOperatorGroups"
+	CSVReasonRequirementsUnknown                         ConditionReason = "RequirementsUnknown"
+	CSVReasonRequirementsNotMet                          ConditionReason = "RequirementsNotMet"
+	CSVReasonRequirementsMet                             ConditionReason = "AllRequirementsMet"
+	CSVReasonOwnerConflict                               ConditionReason = "OwnerConflict"
+	CSVReasonComponentFailed                             ConditionReason = "InstallComponentFailed"
+	CSVReasonInvalidStrategy                             ConditionReason = "InvalidInstallStrategy"
+	CSVReasonWaiting                                     ConditionReason = "InstallWaiting"
+	CSVReasonInstallSuccessful                           ConditionReason = "InstallSucceeded"
+	CSVReasonInstallCheckFailed                          ConditionReason = "InstallCheckFailed"
+	CSVReasonComponentUnhealthy                          ConditionReason = "ComponentUnhealthy"
+	CSVReasonBeingReplaced                               ConditionReason = "BeingReplaced"
+	CSVReasonReplaced                                    ConditionReason = "Replaced"
+	CSVReasonNeedsReinstall                              ConditionReason = "NeedsReinstall"
+	CSVReasonNeedsCertRotation                           ConditionReason = "NeedsCertRotation"
+	CSVReasonAPIServiceResourceIssue                     ConditionReason = "APIServiceResourceIssue"
+	CSVReasonAPIServiceResourcesNeedReinstall            ConditionReason = "APIServiceResourcesNeedReinstall"
+	CSVReasonAPIServiceInstallFailed                     ConditionReason = "APIServiceInstallFailed"
+	CSVReasonCopied                                      ConditionReason = "Copied"
+	CSVReasonInvalidInstallModes                         ConditionReason = "InvalidInstallModes"
+	CSVReasonNoTargetNamespaces                          ConditionReason = "NoTargetNamespaces"
+	CSVReasonUnsupportedOperatorGroup                    ConditionReason = "UnsupportedOperatorGroup"
+	CSVReasonNoOperatorGroup                             ConditionReason = "NoOperatorGroup"
+	CSVReasonTooManyOperatorGroups                       ConditionReason = "TooManyOperatorGroups"
+	CSVReasonInterOperatorGroupOwnerConflict             ConditionReason = "InterOperatorGroupOwnerConflict"
```

Comment:
Two new CSV reasons: `InterOperatorGroupOwnerConflict` and `CannotModifyStaticOperatorGroupProvidedAPIs`